src/components/PatientsTable/PatientsTable.test.tsx

```diff
@@ -0,0 +1,39 @@
+//
+// This source file is part of the Stanford Biodesign Digital Health Spezi Web Health Components open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import { type Patient } from '@medplum/fhirtypes'
+import { render, screen } from '@testing-library/react'
+import { PatientsTable } from '.'
+
+describe('PatientsTable', () => {
+  it('renders element', () => {
+    const onDelete = vitest.fn()
+    const editRoute = '/patients/edit'
+    const data: Patient[] = [
+      { id: '1', name: [{ given: ['Leland'], family: 'Stanford' }] } as Patient,
+    ]
+
+    render(
+      <PatientsTable onDelete={onDelete} editRoute={editRoute} data={data} />,
+    )
+
+    const firstNameHeader = screen.getByText('First Name', {
+      selector: 'button',
+    })
+    expect(firstNameHeader).toBeInTheDocument()
+
+    const lastNameHeader = screen.getByText('Last Name', { selector: 'button' })
+    expect(lastNameHeader).toBeInTheDocument()
+
+    const firstNameCell = screen.getByText('Leland', { selector: 'td' })
+    expect(firstNameCell).toBeInTheDocument()
+
+    const lastNameCell = screen.getByText('Stanford', { selector: 'td' })
+    expect(lastNameCell).toBeInTheDocument()
```

Comment:
I think selectors here do not make much sense. It's an implementation detail of `DataTable` component, it's tested within `DataTable`.